examples/AzureSDKDemoSwift/Source/BlobDownloadViewController.swift

```diff
@@ -56,6 +57,7 @@ class BlobDownloadViewController: UIViewController, MSALInteractiveDelegate {
 
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
+        PHPhotoLibrary.authorizationStatus()
```

Comment:
This is sufficient to fix the permissions issues (assuming permission was previously granted). This was an issue with the client app, not the SDK.

Reply:
Excellente~